docs-chef-io/content/inspec/resources/sshd_active_config.md

```diff
@@ -0,0 +1,99 @@
++++
+title = "sshd_active_config resource"
+draft = false
+gh_repo = "inspec"
+platform = "linux"
+[menu]
+  [menu.inspec]
+    title = "sshd_active_config"
+    identifier = "inspec/resources/os/sshd_active_config.md sshd_active_config resource"
+    parent = "inspec/resources/os"
++++
+Use the `sshd_active_config` Chef InSpec audit resource to find and test configuration data for the OpenSSH daemon. By default, this configuration data is located at `/etc/ssh/sshd_config` on Linux and Unix platforms. However, this resource is designed to retrieve the active configuration file by the sshd process itself. sshd---the OpenSSH daemon---listens on dedicated ports, starts a daemon for each incoming connection, and then handles encryption, authentication, key exchanges, command execution, and data exchanges.
+## Availability
+
+### Install
+
+{{< readfile file="content/inspec/reusable/md/inspec_installation.md" >}}
+
+### Version
+
+This resource first became available in v1.0.0 of InSpec.
```

Comment:
This has been approved by Product and is expected to be in v6.8.0 - you may update the docs.

Reply:
Fixed.